src/main/java/de/srendi/advancedperipherals/common/configuration/PeripheralsConfig.java

```diff
@@ -136,7 +137,8 @@ public PeripheralsConfig() {
 
         enablePlayerDetector = builder.comment("Enable the Player Detector or not.").define("enablePlayerDetector", true);
         playerDetMaxRange = builder.comment("The max range of the player detector functions. If anyone use a higher range, the detector will use this max range. -1 for unlimited").defineInRange("playerDetMaxRange", -1, -1, Integer.MAX_VALUE);
-        playerSpy = builder.comment("Activates the \"getPlayerPos\" function of the Player Detector").define("enablePlayerPosFunction", true);
+        enablePlayerPosFunction = builder.comment("Activates the \"getPlayerPos\" function of the Player Detector").define("enablePlayerPosFunction", true);
+        enablePlayerUUIDFunction = builder.comment("Activates the \"getPlayerUUID\" function of the Player Detector").define("enablePlayerUUIDFunction", true);
```

Comment:
Is there any reason not to get player's UUID but only use their names as identities?

Reply:
UUIDs are guaranteed to be unique, and the UUID can be used to get additional information from the player via EntityAPI.

Reply:
I see, but in fact you can search player's UUID by their name from mojang nameserver, so instead of prevent player to get player's UUID in game, we should set limit at entity API.

Reply:
No need to contact mojang servers if the UUID is already available as a field, no?

Reply:
Yep, so do you think the `enablePlayerUUIDFunction` flag still have any usage?

Reply:
Oh I see your point. What do you mean by setting a limit at the entity API?

Reply:
> Oh I see your point. What do you mean by setting a limit at the entity API?

For example:
```java
if (entity instanceof Player && !config.noPlayerInfo) {
  return playerToLua(xxx)
}
return entityToLua(xxx)
```

Reply:
Ah yes that makes sense, I'll do that.

Reply:
I've replaced `enablePlayerUUIDFunction` with `enablePlayerAccess`, which stops players being used in the EntityAPI. There isn't much point in restricting `playerToLua` because the you're already getting much more information from the `livingEntityToLua` and `entityToLua` calls that will happen anyway.